docs/release-notes/self-hosted/self-hosted-v8.1.0.md

```diff
@@ -0,0 +1,96 @@
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v8.1.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v8.1/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the versions with the Git tag `self-hosted-8.1.0`:
+
+    -   [Codacy Analysis CLI](https://github.com/codacy/codacy-analysis-cli/releases/tag/self-hosted-8.1.0)
+    -   [Codacy Coverage Reporter](https://github.com/codacy/codacy-coverage-reporter/releases/tag/self-hosted-8.1.0)
+
+## Product enhancements
+
+-   Codacy now supports using [expiring access tokens](https://docs.gitlab.com/ee/integration/oauth_provider.html#expiring-access-tokens){: target="_blank"} to integrate with GitLab. Users affected by issues connecting to GitLab should [revoke the GitLab integration on Codacy](https://docs.codacy.com/v8.1/getting-started/which-permissions-does-codacy-need-from-my-account/#revoking-access-to-integrations) and re-login on the Codacy UI with their GitLab accounts. (CY-6117)
+-   The Codacy Coverage Reporter now prints a more explicit error message when using an account API token from a user that [doesn't have permission to upload coverage data](https://docs.codacy.com/v8.1/organizations/roles-and-permissions-for-synced-organizations/). (CY-6084)
+-   Codacy now [displays diff coverage as not applicable](https://docs.codacy.com/v8.1/repositories/pull-requests/#pull-request-quality-overview) (represented by `∅`) when there are no coverable lines included in a pull request, and correctly reports the pull request status on your Git provider in this scenario. (CY-5960)
+
+    ![Not applicable diff coverage](../images/cy-5960.png)
+
+-   [Pull request notification emails](https://docs.codacy.com/v8.1/account/emails/#managing-your-email-notifications) now display the diff coverage metric. (CY-5700)
+
+## Bug fixes
+
+-   Fixed an issue to allow saving coverage data for the first commit in a repository. (CY-6196)
+-   Fixed an issue that could allow an attacker to do destructive actions on a **Codacy personal organization** such as removing members from Codacy or deleting the personal organization on Codacy. The issue didn't allow the attacker to see or change any information on the Git provider organization or repositories. CVSS v3.1 score: 7.9 (High) (CY-6184, CY-6187, CY-6188)
```

Comment:
Not sure how to add information of something like "the attacker would need accounts on the Codacy and provider instance to perform the exploit"? And update the CVSS based on their SH impact (just added that info on the original tickets on JIRA)

Reply:
I updated the release note to include that information:

> Fixed an issue that could allow an attacker **with an account on both the Codacy Self-hosted instance and on the Git provider** to [...]